### PR TITLE
Rename Country to Faction in Player.cs

### PR DESCRIPTION
**Pull request by penev92**
Yet another piece of #7873.
Split into nice small commits for easy reviewing.

### CODE REVIEW

File: OpenRA.Game/Player.cs [hunk 1]
```diff
@@ -193,7 +191,7 @@ public void OnScriptBind(ScriptContext context)
 		public LuaValue Equals(LuaRuntime runtime, LuaValue left, LuaValue right)
 		{
 			Player a, b;
-			if (!left.TryGetClrValue<Player>(out a) || !right.TryGetClrValue<Player>(out b))
```
Comment (Mailaender) [inline OpenRA.Game/Player.cs]: This seems unrelated.

File: OpenRA.Game/Player.cs [hunk 2]
```diff
@@ -193,7 +191,7 @@ public void OnScriptBind(ScriptContext context)
...
+			if (!left.TryGetClrValue(out a) || !right.TryGetClrValue(out b))
```
Comment (Mailaender) [inline OpenRA.Game/Player.cs]: I am just curious why you did this change.

**Comment by Mailaender** [comment]
Looks good. :+1: Seems to be just internal renamings.

**Comment by Mailaender** [comment]
The Lua API also needs an update. https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Mods.Common/Scripting/Properties/PlayerProperties.cs#L34 Probably again keeping the old one printing a note that this will be removed soon.

File: OpenRA.Game/Player.cs (hunk 2 above)
Comment (penev92) [inline OpenRA.Game/Player.cs]: Those were marked as redundant, because the expected type is already specified on the line above: `Player a, b;`.

**Comment by penev92** [comment]
I'll address that in the next PR, as it is an omission from the previous one, not this one.

**Comment by abcdefg30** [comment]
Looks good to me too. :+1: / :white_check_mark:

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/db824811bcabdc6f2cf5c011790022ea678abf0f...9e58de684b24bb02307d6405d1971b3480fed5de)
